### DISCUSSION TITLE
Using two values in a multiplication formula?

### DISCUSSION THREAD

**Discussion by danielrosehill** [Budibase/budibase#14054] [Help]
Context, user engagement form:

![image](https://github.com/Budibase/budibase/assets/40233203/3d8e42e9-cb54-41bb-babd-161cb2789952)

Companies click on a company logo and it sets certain values from a table of financial accounts, like so:

![image](https://github.com/Budibase/budibase/assets/40233203/92c0ff9f-f2e5-4ac7-8818-bc00988de2ac)

Last step in form: users choose a social cost of carbon from one of two options. Upon clicking the button, that value is captured as `coefficientvalue`:

![image](https://github.com/Budibase/budibase/assets/40233203/5bf969b1-2e07-431e-8b92-74592d1de1de)

Final screen displays a few calculated values (emissions scope value multiplied by the selected coefficient).

The stat card will return the individual values at the last stage but not the product of them as denoted like this:

![image](https://github.com/Budibase/budibase/assets/40233203/5770c206-ff7f-45b7-b3e7-2272f7ad6300)

![image](https://github.com/Budibase/budibase/assets/40233203/22b6d9ed-321e-4bed-ba00-d6c8ff181236)

Any help appreciated!

**Comment by ConorWebb96** [comment]
Hey @danielrosehill,

Can you try clearing the white space. Try to leave only 1 space between each `{{ multiply State. scopeltotal_selection State.coefficientvalue }}` alternatively you could just do this in JS `return State. scopeltotal_selection  * State.coefficientvalue`

Hope this helps
